components/landing/TheQuickStart/index.vue

```diff
@@ -2,41 +2,48 @@
   <article class="bx--grid page-section quick-start">
     <h2>Quick Start</h2>
     <div class="bx--row quick-start__introduction">
-      <div class="bx--col-md-5 bx--col-lg-8 bx--col-xlg-7 bx--col-max-6">
-        <p class="quick-start__description">
+      <div class="bx--col-md-8 bx--col-lg-8">
+        <p class="quick-start__introduction__copy">
           When you are looking to start Qiskit, you have two options. You can
           start Qiskit locally, which is much more secure and private, or you
           get started with Jupyter Notebooks hosted in IBM Quantum Lab.
         </p>
+        <p class="quick-start__introduction__copy">
+          Test some of the available providers and algorithms.
+        </p>
+        <ProvidersList :providers-list="providersData" />
       </div>
-    </div>
-    <div class="bx--row">
-      <StartLocally
-        class="bx--col-md-5 bx--col-lg-10 bx--col-xlg-11 bx--col-max-12"
-      />
-      <StartOnTheCloud
-        class="bx--col-md-3 bx--col-lg-6 bx--col-xlg-5 bx--col-max-4"
-      />
+      <Algorithms class="bx--col-md-8 bx--col-lg-8" />
     </div>
   </article>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import rawHardware from '~/content/providers/hardware.json'
+import rawSimulators from '~/content/providers/simulators.json'
+import rawMultiplatforms from '~/content/providers/multiplatforms.json'
 
 @Component
-export default class TheQuickStart extends Vue {}
+export default class TheQuickStart extends Vue {
+  providersData = [
+    ...rawHardware,
+    ...rawSimulators,
+    ...rawMultiplatforms
+  ].map(({ title, websiteCta: { url } }) => ({ title, link: url }));
```

Comment:
If you match the types before using `GeneralLink`, I think you don't need this conversion

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3020/commits/7c7909727e57817c961742721e2c985d837d63b3